index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,900&display=swap');
+        </style>
+
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+
+<link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+      integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap');
+        </style>
+    <title>Visit Rwanda Festivals and Events</title>
+</head>
+<body>
+    <div class="social_icon">
+        <ul>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li><a href="#">English</a>
+                <li><a href="#">My Page</a>
+                    <li><a href="#">Logout</a>
+        </ul>
+    </div>
+    <header id="header">
+        <div class="logo">
+          <a href="index.html">
+          <img src="images/gorilla-logo.png" alt=""></a>
+        </div>
+        <nav id="nav-bar" class="nav-bar d-none">
+          <div id="close-icon" class="close-icon d-none">
+            <i class="fas fa-times-circle"></i>
+          </div>
+          <div class="nav-wrap">
+            <a class="nav-link" id="about" href="about.html" >About</a>
+            <a class="nav-link visited" href="#programs" >Program</a>
+            <a class="nav-link" href="#contents" >Join</a>
+            <a class="nav-link" href="#" >News</a>
+            <a class="nav-link" href="#sponsor" >Sponsor</a>
+            <a class="nav-link" id="home" href="index.html">RW Compaign</a>
+          </div>
+        </nav>
+        <div class="menu">
+          <i class="fas fa-bars"></i>
+        </div>
+      </header>
```

Comment:
- [ ] In the mobile view of this page, I think it will be better if the background image also covers the top space where the hamburger icon is found.